docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/event/Event.scss

```diff
@@ -2,10 +2,8 @@
 @import "~Core/styles/variables";
 @import "~Core/styles/breakpoint-mixins";
 
-.event-location {
-  > span {
-    display: block;
-  }
+p.event-location > span {
```

Comment:
If Event does not exist in ncids_trans you need to copy this file to that location and reference that file. But once again, you cannot modify any files under cgov.